### PR TITLE
00 html css basics

### PR DESCRIPTION
**Pull request by francosili**

### CODE REVIEW

File: 00-HTML-CSS-basics/Exercise-2.5/index.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+	
+			<div class="head">
+				<header>
+					Cine Show Case
+				</header>
+			</div>
+		<div class="grid">
+			<div class="w3-third">
+      
+				<div class="one-third">
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise-2.5/index.html]: Look for this comment and for indentation for this whole file.

File: 00-HTML-CSS-basics/Exercise-2.5/style.css [hunk 2]
```diff
@@ -0,0 +1,117 @@
+.head{
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise-2.5/style.css]: Please fix every declaration standard here. Don't forget the space just before `{`

File: 00-HTML-CSS-basics/Exercise1- Item_6/form.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<!doctype html>	
+
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise1- Item_6/form.html]: Extra line over here.

File: 00-HTML-CSS-basics/Exercise1- Item_6/form.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+<html>
+    <head> 
+		<link type="text/css" rel="stylesheet" href="styles.css" />
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise1- Item_6/form.html]: Look at these indentations

File: 00-HTML-CSS-basics/Exercise1- Item_6/form.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+		<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+	<title>
+	Form
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise1- Item_6/form.html]: This could be in just one line.

File: 00-HTML-CSS-basics/Exercise1- Item_6/form.html [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+	</title>
+		
+	 </head>
+
+
+
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise1- Item_6/form.html]: A lot of extra lines here.

File: 00-HTML-CSS-basics/Exercise1- Item_6/form.html [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+    <body>
+	<form class="contact_form"  id="contact_form" >
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise1- Item_6/form.html]: This is not a convenient class regarding our BEM standards...

File: 00-HTML-CSS-basics/Exercise1- Item_6/form.html [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+		<div>
+			<ul>
+				<li> 
+				<h2>Sing Up Form</h2> 
+				<span class="required_notification">* data required</span>
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise1- Item_6/form.html]: This class is not within our standards... Check for this whole file.

File: 00-HTML-CSS-basics/Exercise1- Item_6/form.html [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+	</title>
+		
+	 </head>
+
+
+
+    <body>
+	<form class="contact_form"  id="contact_form" >
+		<div>
+			<ul>
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise1- Item_6/form.html]: Please, review these indentations.

File: 00-HTML-CSS-basics/Exercise1- Item_6/styles.css [hunk 10]
```diff
@@ -0,0 +1,137 @@
...
+} 
+	
+.contact_form ul {
+	width: 750px; 
+	list-style-type: none; 
+	list-style-position: outside; 
+	 
+	} 
+	
+.contact_form li {
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise1- Item_6/styles.css]: It's not cool to use HTML tags as selector. Please add a class to replace every HTML selector in this file.

File: 00-HTML-CSS-basics/Exercise_1,2,3/HelloWorld.html [hunk 11]
```diff
@@ -0,0 +1,133 @@
+<!doctype html>	
+
+<html>
+    <head> 
+		<link type="text/css" rel="stylesheet" href="styles.css"  media="screen,projection"/>
+		<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+		My Todo List
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise_1,2,3/HelloWorld.html]: This should be wrapped by a `<title>` tag.

File: 00-HTML-CSS-basics/Exercise_1,2,3/HelloWorld.html [hunk 12]
```diff
@@ -0,0 +1,133 @@
...
+		
+	 </head>
+
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise_1,2,3/HelloWorld.html]: Also check for indent and extra lines in this file.

File: 00-HTML-CSS-basics/Exercise_1,2,3/HelloWorld.html [hunk 13]
```diff
@@ -0,0 +1,133 @@
...
+<br>
+<br>
+
+    <body>
+	
+	<div class="days">
+    <table>
+      <tr>
+        <td class="daymonday"><h3>Monday</h3></td>
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise_1,2,3/HelloWorld.html]: It's weird to see a lot of headers inside a table. Even one... You should add style for these instead using any `h` tag.

File: 00-HTML-CSS-basics/Exercise_1,2,3/styles.css [hunk 14]
```diff
@@ -0,0 +1,41 @@
+.days {
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise_1,2,3/styles.css]: Same review from other css files. Missing spaces, fix indentation and no HTML tags.

File: 00-HTML-CSS-basics/Exercise_4/expenses.html [hunk 15]
```diff
@@ -0,0 +1,56 @@
+<!doctype html>	
+
+<html>
+    <head> 
+		<link type="text/css" rel="stylesheet" href="styles.css"  media="screen,projection"/>
+		<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+		
+		Expenses
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise_4/expenses.html]: Title is not wrapped in `<title>`

File: 00-HTML-CSS-basics/Exercise_4/expenses.html [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+		
+	 </head>
+
+<br>
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise_4/expenses.html]: What are doing these `<br>`. They are outside your body.

File: 00-HTML-CSS-basics/Exercise_4/expenses.html [hunk 17]
```diff
@@ -0,0 +1,56 @@
...
+<br>
+
+    <body>
+	
+	
+    <table class="tabler">
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise_4/expenses.html]: Check for indentation and extra spaces here.

File: 00-HTML-CSS-basics/Exercise_4/styles.css [hunk 18]
```diff
@@ -0,0 +1,15 @@
+
+tr:nth-child(even) {
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise_4/styles.css]: Please don't use HTML tag as selector.

**Comment by francosili** [comment]
Ok , I'll make corrections. Thanks for the instructions and i will try to
do better. I dont spent much time because this week I have some subjects
for the University.

File: 00-HTML-CSS-basics/Ejercicio5/media.html [hunk 19]
```diff
@@ -1,10 +1,19 @@
 <!doctype html>	
 <html>
+<<<<<<< HEAD
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Ejercicio5/media.html]: There's a conflict here you've to solve!

File: 00-HTML-CSS-basics/Ejercicio5/media.html [hunk 20]
```diff
@@ -13,6 +22,12 @@
 	<div class="video">
 		<iframe width=”640″ height=”360″ src="https://www.youtube.com/embed/-eGgyREXA9w" rel=0&showinfo=0&controls=1&autoplay=1&start=0” frameborder=”0″ allowfullscreen></iframe>
 	</div>
+<<<<<<< HEAD
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Ejercicio5/media.html]: Please fix this conflict!

File: 00-HTML-CSS-basics/Exercise-2.5/index.html [hunk 21]
```diff
@@ -2,126 +2,95 @@
...
-	<body>
 	
-			<div class="head">
-				<header>
-					Cine Show Case
-				</header>
-			</div>
+    
+<body>
+		<header class="head">Cine Show Case</header>
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise-2.5/index.html]: Fix indentation in this file.

File: 00-HTML-CSS-basics/Exercise-2.5/style.css [hunk 22]
```diff
@@ -1,65 +1,33 @@
...
-.w3-col.s9{width:74.99999%}
-.w3-col.s10{width:83.33333%}
-.w3-col.s11{width:91.66666%}
-.w3-col.s12,.w3-half,.w3-third,.w3-twothird,.w3-threequarter,.w3-quarter{width:99.99999%}
+.w3-third {
+	float:left;
+	width:100%
+}
+
+..w3-third {
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise-2.5/style.css]: You've two points here `.`

File: 00-HTML-CSS-basics/Exercise-2.5/style.css [hunk 23]
```diff
@@ -87,12 +55,13 @@ body {
 	position: absolute;
 	opacity: 0;
 	z-index: 1000;
-  -webkit-transition: all 0.3s ease-in-out;
+	-webkit-transition: all 0.3s ease-in-out;
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise-2.5/style.css]: Weird indentation here.

File: 00-HTML-CSS-basics/Exercise1- Item_6/form.html [hunk 24]
```diff
@@ -1,49 +1,52 @@
...
-
-
+		<title>Form</title>
+	</head>
 
-    <body>
-	<form class="contact_form"  id="contact_form" >
+<body>
+	<form class="contactform">
 		<div>
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise1- Item_6/form.html]: Fix indentation in this file.

File: 00-HTML-CSS-basics/Exercise1- Item_6/form.html [hunk 25]
```diff
@@ -1,49 +1,52 @@
...
+				
+				<li class="contactformli"> 
+					<label>Last Name*</label> 
+					<input type="text" placeholder="Sili" class="inputs" required /> 
 				</li> 
-				<li> 
-				<label>Email</label> 
-				<input type="email" placeholder="fsili72@gmail.com" required /> 
+				
+				<li class="contactformli">
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise1- Item_6/form.html]: This class is pretty weird. Maybe `form--item`. Remember we're using BEM conventions. (And take a look at emails and standards).

File: 00-HTML-CSS-basics/Exercise_1,2,3/HelloWorld.html [hunk 26]
```diff
@@ -1,14 +1,108 @@
 <!doctype html>	
 <html>
+<<<<<<< HEAD
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise_1,2,3/HelloWorld.html]: Here's another conflict.

File: 00-HTML-CSS-basics/Exercise_1,2,3/HelloWorld.html [hunk 27]
```diff
@@ -1,14 +1,108 @@
...
 		<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
 		<title>My Todo List</title>
 	</head>
 
 <body>
+>>>>>>> origin/00-HTML-CSS-basics
 	
+<body>
 	<div class="days">
+<<<<<<< HEAD
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise_1,2,3/HelloWorld.html]: Here's another conflict.

File: 00-HTML-CSS-basics/Exercise_1,2,3/HelloWorld.html [hunk 28]
```diff
@@ -1,14 +1,108 @@
...
+				<td class="day">Monday</td>
+				<td class="day">Tuesday</td>
+				<td class="day">Wednesday</td>
+				<td class="day">Thursday</td>
+				<td class="day">Friday</td>
+			</tr>
+		</table>
+	</div>
+	
+		<table>
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise_1,2,3/HelloWorld.html]: Fix indentation for this file.

File: 00-HTML-CSS-basics/Exercise_1,2,3/styles.css [hunk 29]
```diff
@@ -1,4 +1,25 @@
 .days {
+<<<<<<< HEAD
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise_1,2,3/styles.css]: Please fix this conflict.

File: 00-HTML-CSS-basics/Exercise_4/expenses.html [hunk 30]
```diff
@@ -1,5 +1,54 @@
 <!doctype html>	
 <html>
+<<<<<<< HEAD
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise_4/expenses.html]: Please fix this conflict.

File: 00-HTML-CSS-basics/Exercise_4/expenses.html [hunk 31]
```diff
@@ -1,5 +1,54 @@
...
+		<head> 
+			<link rel="stylesheet" href="styles.css"  />
+			<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+			<title>Expenses</title>
+		</head>
+
+<body>
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise_4/expenses.html]: Fix indentation in this file.

File: 00-HTML-CSS-basics/Exercise_4/styles.css [hunk 32]
```diff
@@ -1,4 +1,19 @@
+<<<<<<< HEAD
```
Comment (elnawe) [inline 00-HTML-CSS-basics/Exercise_4/styles.css]: Please fix this conflict.

**Pull request merged**